source/fundamentals/versioned-api.txt

```diff
@@ -0,0 +1,123 @@
+=============
+Versioned API
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   The Versioned API feature requires MongoDB Server 5.0 or later. You
+   should only use the Versioned API feature if you are connecting to
+   MongoDB servers that are upgraded to a version that supports this feature.
+
+This guide shows you how to specify the **Versioned API** when connecting to
+a MongoDB instance or replica set. You can use the Versioned API feature to
+force the server to run operations with consistent behavior as specified by
+an **API version**. An API version defines expected behavior of operations
+such as the information expected in the request and the response.
+
+When you use the Versioned API feature, you can update your MongoDB driver or
+server without worrying about backward compatibility issues of the commands
+covered by the Versioned API.
+
+..
+   TODO: See the server manual page on `Versioned API <https://docs.mongodb.com/v5.0/reference/versioned-api/>`__
+   for more information including the set the commands covered.
+
+The following sections describe how you can enable Versioned API for
+your MongoDB client and each of the options that you can specify.
+
+Enable Versioned API on a MongoDB Client
+----------------------------------------
+
+To enable the Versioned API, you must specify an API version in the settings
+that you use to create your MongoDB client. Once you instantiate a
+``MongoClient`` instance with a specified Versioned API, all commands you
+run with that client use that version of the Versioned API. If you need to
+run commands not covered by the Versioned API or with a different version of
+it, instantiate a separate client.
+
+The example below shows how you can instantiate a ``MongoClient`` that
+specifies version "1" of the Versioned API and connects to a server by
+performing the following operations:
```

Comment:
Nit: When I didn't see 1 on its own in the snippet, I was a bit confused.
```suggestion
specifies version "V1" of the Versioned API and connect to a server by:
```

Reply:
Good catch. The actual value is "1" whereas the name of the class variable is "V1". I'll add some text to clarify.